packages/backend-core/src/api/endpoints/OrganizationApi.ts

```diff
@@ -160,6 +165,19 @@ export class OrganizationAPI extends AbstractAPI {
     });
   }
 
+  public async updateOrganizationMembershipMetadata(params: UpdateOrganizationMembershipMetadataParams) {
+    const { organizationId, userId, publicMetadata, privateMetadata } = params;
```

Comment:
❓Just asking, If this method is called without providing one of the `publicMetadata` or `privateMetadata` props, eg when a user only wants to update on the of two, the prop not provided will end up as `undefined` here.

Can this lead to unintentionally deleting the corresponding metadata value on the BE?

Reply:
@nikosdouvlis Thanks for raising the concern. The BE API will omit undefined values (as if they are not passed at all). It will work as expected.